operator/src/test/java/oracle/kubernetes/operator/helpers/JobHelperTest.java

```diff
@@ -500,6 +517,57 @@ public void whenDomainHasAdditionalVolumesWithCustomVariablesInvalidValue_jobNot
     assertThat(job, is(nullValue()));
   }
 
+  @Test
+  public void whenDomainHasConfigOverrideSecretsWithLongName_volumeCreatedWithShorterName() {
+    resourceLookup.defineResource(LONG_SECRET_NAME, KubernetesResourceType.Secret, NS);
+
+    configureDomain()
+            .withConfigOverrideSecrets(LONG_SECRET_NAME);
+
+    runCreateJob();
+
+    assertThat(getJobVolumes(), hasSecretVolume(VOLUME_NAME_FOR_LONG_SECRET_NAME, LONG_SECRET_NAME, DEFAULT_MODE));
+    assertThat(getJobVolumeMounts(), hasVolumeMount(VOLUME_NAME_FOR_LONG_SECRET_NAME,
+            "/weblogic-operator/config-overrides-secrets/" + LONG_SECRET_NAME, true));
+  }
+
+  @Test
+  public void whenDomainHasMultipleConfigOverrideSecretsWithLongNames_volumeCreatedWithShorterNames() {
+    resourceLookup.defineResource(LONG_SECRET_NAME, KubernetesResourceType.Secret, NS);
+    resourceLookup.defineResource(SECOND_LONG_SECRET_NAME, KubernetesResourceType.Secret, NS);
```

Comment:
Probably want to add multiple long named secrets where the first 63 characters are the same.

Reply:
Done